File: Iniciante/URI 1009.cs
using System; 

class URI {

    public static void Main(string[] args) { 

        string nome;
        double salario, montante, lucro, total;
        
        nome = Console.ReadLine();
        
        salario = Double.Parse(Console.ReadLine());
        montante = Double.Parse(Console.ReadLine());
        
        lucro = 0.15 * montante;
        
        total = salario + lucro;
        
        Console.WriteLine("TOTAL = R$ {0:F2}", total);
        
    }
}

File: Iniciante/URI 1009.cpp
#include <stdio.h>

int main()
{

  char nome[21];
  double SALARIO, MONTANTE, lucro, TOTAL;

  scanf("%s", &nome);
  scanf("%lf %lf", &SALARIO, &MONTANTE);

  lucro = 0.15 * MONTANTE;
  TOTAL = SALARIO + lucro;

  printf("TOTAL = R$ %.2lf\n", TOTAL);

  return 0;
}